<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
    <item>
     <layout class="QVBoxLayout" name="ctrl_layout" stretch="0,0,0,0,0,0">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Loc pos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="loc_pos">
          <item>
           <property name="text">
            <string>top right</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>center right</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>center left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>top left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bottom right</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bottom left</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>纵轴范围:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>L:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_YL">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>H:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_YH">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="redraw_button">
          <property name="text">
           <string>刷新绘制</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="add_item_button">
          <property name="text">
           <string>添加选项:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_itemString"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1,3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>可选选项</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_selects"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="add_all">
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="del_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="del_all">
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>显示选项</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_selected"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="matplot_layout"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionSave_windows"/>
   </widget>
   <widget class="QMenu" name="menu_Ctrl">
    <property name="title">
     <string>&amp;Ctrl</string>
    </property>
    <addaction name="action_Show_Ctrl_layout"/>
    <addaction name="action_Hide_Ctrl_layout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Ctrl"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Show_Ctrl_layout">
   <property name="text">
    <string>&amp;Show_Ctrl_layout</string>
   </property>
  </action>
  <action name="action_Hide_Ctrl_layout">
   <property name="text">
    <string>&amp;Hide_Ctrl_layout</string>
   </property>
  </action>
  <action name="actionSave_windows">
   <property name="text">
    <string>&amp;Save windows</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_button_onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>del_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>del_button_onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_selects</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>listWidget_selects_itemDoubleClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_selected</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>listWidget_selected_itemDoubleClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redraw_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>update_draw()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>del_all</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>del_all_button_onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_all</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_all_button_onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_item_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_item_button_onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_button_onClick()</slot>
  <slot>del_button_onClick()</slot>
  <slot>listWidget_selected_itemDoubleClicked()</slot>
  <slot>listWidget_selects_itemDoubleClicked()</slot>
  <slot>update_draw()</slot>
  <slot>add_all_button_onClick()</slot>
  <slot>del_all_button_onClick()</slot>
  <slot>add_item_button_onClick()</slot>
  <slot>open_file_button_onClick()</slot>
 </slots>
</ui>
